ds_judgements_public_ui/templates/includes/courts_options.html

```diff
@@ -7,7 +8,9 @@
              id="court_{{ court.canonical_param }}"
              {% for alias in context.court %}{% if alias in court.param_aliases %}checked="checked"{% endif %}
              {% endfor %} />
-      <span class="structured-search__court-option-label">{{ court.name }}</span>
+      <span class="structured-search__court-option-label">{{ court.name }}
+        <span style="color:#555; font-size: 0.9rem">&mdash; {{ court|get_court_date_range }}</span>
```

Comment:
could we add this style in the appropriate SCSS file (`_structured_search.scss`) and add a class here (following BEM conventions, something like `structured-search__court-date-range` would be appropriate) , rather than using inline styles?

 It'd also be good to use a named colour variable (that one is $color__dark-grey) rather than hardcoding the colour

Reply:
@timcowlishaw  all done and added the image for results filter